tests/integration/test_interface.py

```diff
@@ -809,9 +811,10 @@ def _validate_column_noise_level(
 
             # Calculate probability no token is noised
             if col_noise_type.name == NOISE_TYPES.write_wrong_zipcode_digits.name:
+                token_probabilities: list[float] = token_probability
```

Comment:
A better solution here would be to `if isinstance(token_probability, list)`:... instead of checkking the name of the noise type